### DISCUSSION TITLE
How to use the prettier plugin for pug

### DISCUSSION THREAD

**Discussion by skandasoft** [prettier/plugin-pug#297] [Q&A]
How to use the prettier plugin for pug..it doesn't seem to work?
![image](https://user-images.githubusercontent.com/9875379/135345384-406806fa-70aa-417f-a9c6-ccaf22bac8c0.png)
I installed the prettier/@prettier-plugin-pug

**Comment by skandasoft** [comment]
My Prettier File is .prettierrc
```
{
  "singleQuote": true,
  "overrides": [
    {
      "files": "*.pug",
      "options": {
        "parser": "pug",
        "singleQuote": true
      }
    }
  ]
}

```

and package.json

```
  "devDependencies": {
    "@prettier/plugin-pug": "^1.16.7",
    "prettier": "^2.4.1",    
```

**Reply by Shinigami92** [reply]
So I assume you didn't installed `prettier/@prettier-plugin-pug` but `@prettier/plugin-pug` like you showed in your part of the `package.json`

Also it is correct to install `prettier`
This was also mentioned in the https://github.com/prettier/plugin-pug#getting-started section

You do not get any log outputs?
It should work out of the box without the need of a `.prettierrc` file, so you should test it first without this file and then start to configure and debug your way to success.

Without any more info I cannot really help you, cause it must be an issue on your side. Cause it works in all other known projects.

**Reply by Shinigami92** [reply]
Maybe you need to enable the formatter for pug to be [esbenp.prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
But this is only for VSCode and doesn't have something to do with this plugin itself 🤔
